src/server/table_stats.h

```diff
@@ -116,6 +123,9 @@ struct table_stats
     double total_check_and_set_qps = 0;
     double total_check_and_mutate_qps = 0;
     double total_scan_qps = 0;
+    double total_duplicate_qps = 0;
```

Comment:
Should duplicate_qps be showed in `app_stat -q`?

Reply:
I think showing it on perf-counter is enough for now. Because most tables do not need duplication. I may show it on another shell command like query_dup if needed.